src/suite-design-guide/portable-suites.rst

```diff
@@ -334,10 +329,7 @@ and batch scheduler directives.
        [[root]]
            script = rose task-run -v
        [[HPC]]  # NIWA job host and batch scheduler settings.
-           [[[remote]]]
-               host = hpc1.niwa.co.nz
-           [[[job]]]
-               batch system = loadleveler
+           platform = niwa_loadleveler_platform
            [[[directives]]]
                account_no = NWP1623
                class = General
```

Comment:
Oops, still a reference to batch systems just before this code block
>```
>The remaining tasks are site-agnostic apart from local job host
>and batch scheduler directives.
>```

Reply:
Well spotted

Reply:
(It was me who missed it the last time round!)